### PR TITLE
FH-1652 : add push plugin by default

### PR DESCRIPTION
**Pull request by sebastienblanc**
@edewit @jasonmadigan Could you check if this is correct ?

### CODE REVIEW

File: www/config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "plugins": [
+    {
+      "id": "org.jboss.aerogear.cordova.push",
+      "version": "2.x-compat",
```
Comment (edewit) [inline www/config.json]: version should be 2.0.2-compat

**Comment by sebastienblanc** [comment]
@edewit should be good now ;)

**Comment by jasonmadigan** [comment]
@sebastienblanc looks good

to update templates in fhcap:

* merge this
* Bump the version in https://github.com/feedhenry/template-config, create a PR - this will build an archive of all templates (and will include this change)
* Bump the build number of fh-template-apps in fhcap

**Comment by sebastienblanc** [comment]
@jasonmadigan looks like I don't have push permission on this repo, are you able to give me push rights ? or @johnfriz ?

**Comment by matzew** [comment]
same here - no write access

**Comment by jasonmadigan** [comment]
@matzew @sebastienblanc invites to owners sent.

**Pull request merged**